tracer/transport.go

```diff
@@ -22,6 +22,12 @@ type Transport interface {
 	SetHeader(key, value string)
 }
 
+// NewHTTPTransport returns a new Transport implementation using the given
+// delivery url.
+func NewHTTPTransport(url string) Transport {
```

Comment:
I think #17 would definitely be useful, but would also like an API for configuring this outside of environment variables, so it can be integrated with existing tools that might use `--flags` or config files.